RZViewActions/UIView+RZViewActions.m

```diff
@@ -72,30 +77,28 @@ + (void)rz_runAction:(RZViewAction *)action withCompletion:(RZViewActionCompleti
 
 @implementation RZViewAction
 
-- (instancetype)initWithBlock:(RZViewActionBlock)block options:(UIViewAnimationOptions)options duration:(NSTimeInterval)duration
+#pragma mark - public methods
+
++ (RZViewAction *)action:(RZViewActionBlock)action withDuration:(NSTimeInterval)duration
 {
-    self = [super init];
-    if ( self ) {
-        _block = block;
-        _options = options;
-        _duration = duration;
-    }
-    return self;
+    return [self action:action withOptions:kNilOptions duration:duration];
 }
 
-+ (instancetype)action:(RZViewActionBlock)action withDuration:(NSTimeInterval)duration
++ (RZViewAction *)action:(RZViewActionBlock)action withOptions:(UIViewAnimationOptions)options duration:(NSTimeInterval)duration
 {
-    return [self action:action withOptions:kNilOptions duration:duration];
+    return [[RZViewAction alloc] initWithBlock:action options:options duration:duration];
```

Comment:
Why was this changed from `self alloc`?  if this is subclassable, we are better off using 'self'